conformance/tests/overloads_definitions.py

```diff
@@ -70,28 +70,28 @@ def not_abstract(self, x: str) -> str: ...
 class C:
     @overload  # E[func5]
     @staticmethod
-    def func5(x: int) -> int:  # E[func5]
+    def func5(x: int, /) -> int:  # E[func5]
```

Comment:
I tried to get rid of various unrelated errors detected by some type checkers so that this test case focuses on only the issue it's trying to test for.